tests/test_fuzzy_dedup.py

```diff
@@ -338,7 +338,7 @@ def test_fuzzy_dedup(
             num_anchors=2,
             jaccard_threshold=jaccard_threshold,
         )
-        fuzzy_duplicates = FuzzyDuplicates(config=config)
+        fuzzy_duplicates = FuzzyDuplicates(config=config, perform_removal=False)
```

Comment:
Can we also add a test for perform_removal=True somewhere just to be safe.

Reply:
Added